### DISCUSSION TITLE
Remove q-loading-bar

### DISCUSSION THREAD

**Discussion by ncamaa** [quasarframework/quasar#12757] [General - Components / Directives / etc]
I've been developing an app with Quasar v1 for a while, and I wasn't able to figure out how to remove the `q-loading-bar` element. Any clues?

**Comment by pdanpdan** [comment]
The first yellow box https://v1.quasar.dev/quasar-plugins/loading-bar#installation when using UMD?
When using Quasar CLI put the same setting in the defaults https://v1.quasar.dev/quasar-plugins/loading-bar#setting-up-defaults

**Reply by ncamaa** [reply]
@pdanpdan - thanks for your reply.
I am using Quasar CLI and still can't completely understand. I didn't add it in `quasar.conf.js` but it still shows. I don't want to disable it after it is being loaded, I want to prevent it from loading in the first place.

**Comment by pdanpdan** [comment]
It should only load if you have 
`framework.importStrategy === 'all' || framework.plugins.includes('LoadingBar')`
in quasar.conf.js

**Reply by ncamaa** [reply]
It is true, I do have `framework.importStrategy === 'all'` 
How can I tell it to keep importing whatever it imported so far but the loading bar (without the loading bar)

**Comment by pdanpdan** [comment]
framework.importStrategy === 'auto'

**Reply by ncamaa** [reply]
Thanks. It worked 👍
